code/drasil-metadata/lib/Data/Drasil/DrasilConfig.hs

```diff
@@ -10,10 +10,22 @@ import Utils.Drasil(drasilConfigPath)
 
 newtype DrasilConfig = DrasilConfig {version :: String} deriving Generic
 
+-- The number of folders that needs to be exited in order to obtain the
+-- DrasilConfiguration.json file
+defaultFolderVal = 3
+
 instance FromJSON DrasilConfig
 
-obtainVersion :: Int -> IO String
-obtainVersion num = do
+-- Reads the DrasilConfiguration.json file and extracts the data from that file.
+obtainVersion :: Maybe Int -> IO String
+obtainVersion Nothing = do
+  input <- B.readFile $ drasilConfigPath defaultFolderVal
+  let dc = decode input :: Maybe DrasilConfig
+  case dc of
+    Nothing -> return "error parsing JSON"
+    Just d -> return ((show.version) d)
+
+obtainVersion (Just num) = do
```

Comment:
This entire case (including the parts I can't select because they weren't changed in this PR) should be removed after using `fromMaybe` above